src/servers/trigger.ts

```diff
@@ -255,7 +261,13 @@ const main = () => {
 const healthCheck = () => {
   const app = express()
   const port = 8888
-  app.get("/healthz", (req, res) => res.sendStatus(200))
+  app.get("/healthz", async (_req, res) => {
+    const isMongoAlive = mongoose.connection.readyState === 1
+    const isRedisAlive = (await redis.ping()) === "PONG"
+    const statuses = Object.values(lndStatus)
+    const areLndsAlive = statuses.length > 0 && statuses.every((s) => s)
```

Comment:
I'm not sure to understand the `statuses.every((s) => s)`. It's ok for 1 lnd to be offline. this is the idea behind the high availability that we're working on. I think the code here assumes that all lnd needs to be online?